secure/key-manager/src/main.rs

```diff
@@ -36,6 +36,8 @@ fn main() {
         .is_async(key_manager_config.logger.is_async)
         .level(key_manager_config.logger.level)
         .init();
+    libra_logger::init_struct_log_from_env().expect("Failed to initialize structured logging");
```

Comment:
should be easy to add support for logging to a local file as well. https://github.com/libra/libra/blob/master/common/logger/src/struct_log.rs#L185
sometimes log to local file is easier to use.

Reply:
Thanks, @sherry-x! Done :)